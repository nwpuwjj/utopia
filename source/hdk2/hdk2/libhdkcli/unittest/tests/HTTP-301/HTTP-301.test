#
# HTTP-301.test
#   Verify libhdkcli handles a HTTP 301 server response.
#

method: http://cisco.com/Unittest/SimpleUnittestMethod
platform-expected