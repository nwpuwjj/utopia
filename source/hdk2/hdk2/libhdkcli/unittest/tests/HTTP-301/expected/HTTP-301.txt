+++++++ Request Headers +++++++
HTTP/1.1 POST /HNAP1
Connection: Close
Content-Type: text/xml; charset="utf-8"
Content-Length: 316
Authorization: Basic Og==
SOAPAction: "http://cisco.com/Unittest/SimpleUnittestMethod"
Host: localhost:8080


------- Response -------
HTTP/1.1 301 Moved Permanently
Connection: close
Location: http://localhost:8080/Redirect

+++++++ Request Headers +++++++
HTTP/1.1 GET /Redirect
Connection: Close
Content-Type: text/xml; charset="utf-8"
Authorization: Basic Og==
SOAPAction: "http://cisco.com/Unittest/SimpleUnittestMethod"
Host: localhost:8080


------- Response -------
HTTP/1.1 200 OK
Content-Type: text/xml; charset="utf-8"
Connection: close
Content-Length: 321

<?xml version="1.0" encoding="utf-8" ?>
<soap:Envelope xmlns:soap="http://schemas.xmlsoap.org/soap/envelope/">
<soap:Body>
  <SimpleUnittestMethodResponse xmlns="http://cisco.com/Unittest/">
      <SimpleUnittestMethodResult>OK</SimpleUnittestMethodResult>
  </SimpleUnittestMethodResponse>
</soap:Body>
</soap:Envelope>


******* Client Result ******

      SOAP action: http://cisco.com/Unittest/SimpleUnittestMethod
        HTTP host: http://localhost:8080
Network Object ID: (null)
    HTTP username: (null)
    HTTP password: (null)
          timeout: 0
       HNAP input: HTTP-301.input.xml

<?xml version="1.0" encoding="utf-8"?>
<Envelope xmlns="http://schemas.xmlsoap.org/soap/envelope/">
<Body>
<SimpleUnittestMethodResponse xmlns="http://cisco.com/Unittest/">
<SimpleUnittestMethodResult>OK</SimpleUnittestMethodResult>
</SimpleUnittestMethodResponse>
</Body>
</Envelope>
malloc_interposer.c - ************************************************************
malloc_interposer.c - exit: malloc called 78 times
malloc_interposer.c - exit: free called 78 times
malloc_interposer.c - exit: malloc current = 0
malloc_interposer.c - exit: malloc total = 18642
malloc_interposer.c - exit: malloc max = 11168
malloc_interposer.c - ************************************************************
